### PR TITLE
Make go generate's output reproducible

### PR DESCRIPTION
**Pull request by mallorydxw**
This will prevent unnecessary changes to `raw_templates.go` by using a
sorted list of keys instead of relying on the default ordering of the
map (which is random in Go).

### CODE REVIEW

File: raw_templates.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...ins-to-activate\"\ncontent=/usr/src/app/setup/content\n\nwp core {{.InstallType}} --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=\"$title\"\n\nfor plugin in $plugins\ndo\n  if wp plugin is-installed $plugin\n  then\n    wp plugin activate $plugin {{.ActivationType}}\n  else\n      echo \"\\033[96mWarning:\\033[0m Plugin '\"$plugin\"' could not be found. Have you installed it?\"\n  fi\ndone\n\nif wp theme is-installed $theme\nthen\n  {{.ThemeEnable}}\n  wp theme activate $theme\nelse\n  echo \"\\033[96mWarning:\\033[0m Theme '\"$theme\"' could not be found. Have you installed it?\"\nfi\n\nimport() {\n  for file in $content/*.xml\n  do\n    echo \"Importing $file...\"\n    wp import $file --authors=skip\n  done\n}\n\nif [ \"$(ls -A $content)\" ]\nthen\n  if wp plugin is-installed wordpress-importer\n  then\n    wp plugin activate wordpress-importer\n    import\n  else\n    echo \"WordPress Importer not installed... installing now\"\n    wp plugin install wordpress-importer --activate\n    import\n    wp plugin uninstall wordpress-importer --deactivate\n  fi\nelse\n  echo \"No content to be imported\"\nfi\n"}
```
Comment (erbridge) [inline raw_templates.go]: Do we need to be committing this at all? Can we generate this file at runtime?

Comment (mallorydxw) [reply raw_templates.go]: I assume you mean at install/build time?

If we did that it wouldn't be possible to install with `go get github.com/dxw/wpc`.

Generated files tend to be part of the repo anyway. For example `.c` (C) files that have been generated from `.yy` (yacc) files.

Comment (erbridge) [reply raw_templates.go]: Ah. Ok. Let's label it as generated using a comment, then?

Comment (mallorydxw) [reply raw_templates.go]: @erbridge Pushed another commit. Ready for another look.

File: raw_templates.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
 package main
-
+// This file is automatically generated by generate.go
+// Do not edit manually
```
Comment (erbridge) [inline raw_templates.go:3]: Shouldn't this be at the top of the file? Isn't that how `godoc` works?

Comment (erbridge) [reply raw_templates.go:3]: I thought `go generate` generated these for you, also: https://blog.golang.org/generate

Comment (mallorydxw) [reply raw_templates.go:3]: You would put the documentation comment above the `package main` line if you wanted that documentation to be added to the whole package. You can demonstrate this by moving the `package main` line to below the comments and running `go doc` which reveals the documentation for the current package.

But `main` packages don't really need documentation in the same way other packages do, so we're not too worried about the content of `go doc`'s output.

**Comment by mallorydxw** [comment]
I'll merge this one too, without approval. Because it's been open a while and we should probably put these changes into `whippet-template`.

**Pull request merged**